setup.cfg

```diff
@@ -28,10 +28,11 @@ classifiers =
 
 [options]
 install_requires =
-    boto3
-    boto3-stubs[apigateway,cognito,essential]
+    boto3~=1.25
+    boto3-stubs[apigateway,cognito,essential]~=1.25
+    joblib~=1.3.1
     requests>=2.23
-    types-requests
+    types-requests>=2.23
```

Comment:
You probably want `~=2.23` to avoid the dependency confusion vulnerability here. same for `requests>=2.23` → `requests~=2.23`.

(Example case is: someone hacks `requestst` on PyPI and publishes a malicious version `999.99.99` which gets pulled in.)